tests/TestDataService.spec.ts

```diff
@@ -23,13 +23,51 @@ test('Data Service', async ({
     expect(customer.firstName).toEqual('Luke');
     expect(customer.lastName).toEqual('Skywalker');
 
+    const customerAddress = await TestDataService.getCustomerAddress(customer.defaultBillingAddressId);
+    const deliveryStateMachine = await TestDataService.getDeliveryStateMachine();
+    const deliveryState = await TestDataService.getStateMachineState(deliveryStateMachine.id);
+    const shippingMethod = await TestDataService.getShippingMethod();
+    const deliveryStruct = TestDataService.getBasicOrderDeliveryStruct(deliveryState, shippingMethod, customerAddress);
+    const shippingCosts = 5;
+    const totalPrice = 50+ shippingCosts;
+    if (deliveryStruct.shippingCosts != null) {
+        deliveryStruct.shippingCosts.unitPrice = shippingCosts;
+        deliveryStruct.shippingCosts.totalPrice = shippingCosts;
+    }
+
     const order = await TestDataService.createOrder(
         [{ product, quantity: 5 }],
         customer,
-        { orderNumber: '123456789' });
-    expect(order.orderNumber).toEqual('123456789');
-    expect(order.orderCustomer.firstName).toEqual('Luke');
-    expect(order.price.totalPrice).toEqual(58.99);
+        {
```

Comment:
I would define the overrides as a separate const and pass it here to make it more readable.